arches_lingo/templates/arches_urls.htm

```diff
@@ -16,6 +16,7 @@
     api_scheme_label_create="{% url 'api-scheme-label-create' %}"
     api_scheme_note='(resourceid)=>{return "{% url "api-scheme-note" "aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa" %}".replace("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa", resourceid)}'
     api_scheme_note_tile='(resourceid, tileid)=>{return "{% url "api-scheme-note-tile" "aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa" "aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaab" %}".replace("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa", resourceid).replace("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaab", tileid)}'
+    api_scheme_note_create="{% url 'api-scheme-note-create' %}"
```

Comment:
should this just fall under the api_scheme_note as a different verb?

Reply:
It looks like it, but that doesn't actually work.  The difference is in pythonic_models.py - you'll see there are different views used for these routes.  The ListCreateView is necessary here because we're dealing with a list of tiles.  If I was to try a patch or a put against api_scheme_note (which uses RetrieveUpdateDestroyAPIView) I'd blow away the rest of the list.